topic-operator/src/main/java/io/strimzi/operator/topic/Session.java

```diff
@@ -223,6 +224,12 @@ public void handle(Long oldTimerId) {
                             timerId = null;
                             boolean isInitialReconcile = oldTimerId == null;
                             topicOperator.reconcileAllTopics(isInitialReconcile ? "initial " : "periodic ").setHandler(result -> {
+                                topicOperator.getPeriodicReconciliationsCounter().increment();
+                                if (result.failed()) {
+                                    topicOperator.incrementFailedReconciliationsCounter();
```

Comment:
This looks wrong to me -> In the other operators:
* Every periodic reconciliation increases the periodic counter + triggers reconciliation for every resource (topic in this case)
* Every reconcilition for a prticular topic triggers its own counter
    * The sucess and failure counters as well as timers are counted only against the particular topics reconciliation. Not against the periodic reconciliations. So I don't think you should increment here the failed and success counters.